skills/ship/epic-runner/SKILL.md

```
@@ -129,6 +138,7 @@ concrete commands for five operations:
 | `start` | Mark an issue as being worked (optional) |
 | `comment` | Post a comment on an issue |
 | `close` | Explicitly mark an issue complete |
+| `create` | File a new issue (only for `--new-issues=create|propose`) |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)